src/main/java/amazon/AuthorSearchPage.java

```diff
@@ -3,29 +3,52 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
+import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 import java.util.List;
 
 public class AuthorSearchPage {
 
-    private WebDriver driver;
+    private final WebDriver driver;
+
+    private static final String searchResultsXpathContent = "//*[@data-component-type='s-search-result']";
+    @FindBy(xpath = searchResultsXpathContent)
+    WebElement searchResults;
+
+
+    private static final String countOfSearchResultsOnFirstPagXpathContent = "//*[@class='a-section a-spacing-small a-spacing-top-small']/span";
```

Comment:
Just to keep `final` ?

Reply:
No, I ask why we need this variable?

Reply:
I applied `static final` to locators, except one, for annotating their `CONSTANT` stage.
Though now I see `final` will be fine, right?

Reply:
You've used it only in one place, where except constant nothing can be accepted - inside @FindBy annotation.
So why just not use the locator in @FindBy instead of keeping it in a separate variable and then use the variable in @FindBy?

Reply:
For readability, I thought keeping it in a variable might be helpful.

Reply:
That's right and therefore your WebElement's name should be meaningful :)

Reply:
Addressed in dc58bf0.